addons/mail/static/src/js/composers/mention_manager.js

```diff
@@ -181,7 +181,13 @@ var MentionManager = Widget.extend({
      */
     propositionNavigation: function (keycode) {
         var $active = this.$('.o_mention_proposition.active');
-        if (keycode === $.ui.keyCode.ENTER) {
+        if (this._activeListener.delimiter === ':' && !$active.length) {
+            if (_.contains([$.ui.keyCode.DOWN, $.ui.keyCode.TAB], keycode)) {
```

Comment:
@pka-odoo  We have already condition based on all keycodes [see](https://github.com/odoo/odoo/blob/master/addons/mail/static/src/js/composers/mention_manager.js#L190) so you can manage inside that. No need to add an extra condition here with all these keycode conditions. You can take here $active as this.$('.o_mention_proposition') because now we don't have active class for ':' delimiter.

Reply:
@hbh-odoo 
Yes, I know we can manage in the existing conditions, but then we have only this way

```
	if (_.contains([$.ui.keyCode.DOWN, $.ui.keyCode.TAB], keycode)) {
	    if ($active.length) {
	        $to = $active.nextAll('.o_mention_proposition').first();
	        if (!$to.length && keycode === $.ui.keyCode.TAB) {
	            $to = $active.prevAll('.o_mention_proposition').last();
	        }
	    } else {
	        $to = this.$('.o_mention_proposition').first();
	    }
	} else if (keycode === $.ui.keyCode.UP) {
	    if ($active.length) {
	        $to = $active.prevAll('.o_mention_proposition').first();
	    } else {
	        $to = this.$('.o_mention_proposition').last();
	    }
	}
```

Is it ok for you? (and here we have a condition of not having active class, as we pretend that it will be not added for delimiter : only)


For this -> $active as this.$('.o_mention_proposition') because now we don't have an active class for ':' delimiter ->  we can't do this, as this method is generic for all the delimiters.
This commit (https://github.com/odoo/odoo/pull/46825/commits/614dde6cfb539ad008300c8809f30d5b3c9af7fd) is done for all the possible cases.

Reply:
@pka-odoo Please look at [commit](https://github.com/odoo-dev/odoo/commit/8ac9d053ff43601151fdc8c9b51397ba4da514ba) This is what i am trying to say. This way we can avoid putting some extra stuff specifically regarding keycodes.
Thanks!